arch/Kconfig

```diff
@@ -34,8 +34,10 @@ config ARM
 	# is really only necessary for Cortex-M with ARM MPU!
 	select GEN_PRIV_STACKS
 	select ARCH_HAS_THREAD_LOCAL_STORAGE if CPU_CORTEX_R || CPU_CORTEX_M
+	select USE_SWITCH_SUPPORTED if CPU_AARCH32_CORTEX_A
```

Comment:
Is there a difference between defining:
- `select USE_SWITCH_SUPPORTED if CPU_AARCH32_CORTEX_A` here 
- `select USE_SWITCH_SUPPORTED` in the [`CPU_AARCH32_CORTEX_A`](https://github.com/zephyrproject-rtos/zephyr/blob/main/arch/arm/core/aarch32/Kconfig#L49-L58) definition?

Reply:
I think no difference, but the 2nd one is better